sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_common/onedp/operations/_operations.py

```diff
@@ -2101,8 +2101,10 @@ def operation_results(self, operation_id: str, **kwargs: Any) -> List[Dict[str,
 
         if _stream:
             deserialized = response.iter_bytes()
-        else:
+        elif type(response.json()) == list:

```

Comment:
Did you have to make this change manually ?

Reply:
Yes. get_operation_results doesn't currently exist on redteamoperations so we are reusing the one in evaluationoperations for now. cc @posaninagendra